code/__DEFINES/achievements.dm

```diff
@@ -37,6 +37,7 @@
 #define MEDAL_TRAM_SURFER "Tram Surfer"
 #define MEDAL_CULT_SHUTTLE_OMFG "WHAT JUST HAPPENED"
 #define MEDAL_CLICKBAIT "Clickbait"
+#define MEDAL_SPRINGLOCK "Springlock"
```

Comment:
this should probably be the same name as the achievement name

Reply:
Alright I’ll do some name tinkering tomorrow